source/github-helpers/get-user-avatar.ts

```diff
@@ -0,0 +1,21 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+export default function getUserAvatar(username: string, size: number): string | void {
```

Comment:
Well I am going to review. 🤣 

Can you please change this to an object {username, size}?
So `getUserAvatar(username, 16);` has a description.

Reply:
The variable names are obvious enough I think. Or just look at the function signature.